### PR TITLE
Update livecontainers.md

### PR DESCRIPTION
**Pull request by nammn**
### Preview

https://docs-staging.datadoghq.com/nammn-patch-1/infrastructure/livecontainers

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/infrastructure/livecontainers.md [hunk 1]
```diff
@@ -87,7 +87,22 @@ In some setups, the Process Agent and Cluster Agent are unable to automatically
...
+Live Containers need permissions to collect common kubernetes resources (pods, services, nodes ..), which should be already in the RBAC if following [Cluster Agent Setup documentation][2].
+If not, the missing resources should be added:
+```
+          - apiGroups:  # to collect new resource types
+            - "apps"
+            resources:
+            - deployments
+            - replicasets
+            - pods
+            - nodes
```
Comment (mfpierre) [inline content/en/infrastructure/livecontainers.md:99]: shouldn't it be added to the above block?

Comment (nammn) [reply content/en/infrastructure/livecontainers.md:99]: hmm, yes true. I just wanted to make clear that they only need to add this part, if one did not follow the typical DCA setup. 🤔

Comment (kayayarai) [reply content/en/infrastructure/livecontainers.md:99]: Maybe we should add this instruction in line 60, something like: 
```
    - Set the Cluster Agent ClusterRole with the following RBAC permissions. Note particularly that for the `apps` apiGroups, Live Containers need permissions to collect common kubernetes resources (`pods`, `services`, `nodes`, etc.), which should be already in the RBAC if you followed [Cluster Agent Setup documentation][2]. But if they are missing, ensure they are added (after `deployments`, `replicasets`, etc.):
```
... and then add `pods` `nodes` etc after line 84. 

What do you think?

Comment (nammn) [reply content/en/infrastructure/livecontainers.md:91]: This sounds great! I added this + a comment to point that pods etc. only needs to be added if missing

**Review by apigirl — APPROVED** [review]

**Pull request merged**